packages/data/src/world/mm/great_bay_temple.yml

```diff
@@ -179,14 +179,14 @@
-    "Great Bay Temple Pot Green Pipe 2 2": "(can_dive_small && (has_weapon_range || has_bombchu)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && short_hook_anywhere)"
-    "Great Bay Temple Pot Green Pipe 2 3": "(can_dive_small && (has_weapon_range || has_bombchu)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && short_hook_anywhere)"
-    "Great Bay Temple Pot Green Pipe 2 4": "(can_dive_small && (has_weapon_range || has_bombchu)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && short_hook_anywhere)"
-    "Great Bay Temple Pot Green Pipe 2 5": "(can_dive_small && (has_weapon_range || has_bombchu)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && short_hook_anywhere)"
-    "Great Bay Temple Pot Green Pipe 2 6": "(can_dive_small && (has_weapon_range || has_bombchu)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && short_hook_anywhere)"
-    "Great Bay Temple Pot Green Pipe 2 7": "(can_dive_small && (has_weapon_range || has_bombchu)) || underwater_walking || (can_use_ice_arrows && short_hook_anywhere)"
-    "Great Bay Temple Pot Green Pipe 2 8": "(can_dive_small && (has_weapon_range || has_bombchu)) || underwater_walking || (can_use_ice_arrows && short_hook_anywhere)"
+    "Great Bay Temple Pot Green Pipe 2 1": "(can_dive_small && (has_arrows || has_bombchu || can_hookshot)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && (short_hook_anywhere || can_dive_small))"
+    "Great Bay Temple Pot Green Pipe 2 2": "(can_dive_small && (has_arrows || has_bombchu || can_hookshot)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && (short_hook_anywhere || can_dive_small))"
+    "Great Bay Temple Pot Green Pipe 2 3": "(can_dive_small && (has_arrows || has_bombchu || can_hookshot)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && (short_hook_anywhere || can_dive_small))"
+    "Great Bay Temple Pot Green Pipe 2 4": "(can_dive_small && (has_arrows || has_bombchu || can_hookshot)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && (short_hook_anywhere || can_dive_small))"
+    "Great Bay Temple Pot Green Pipe 2 5": "(can_dive_small && (has_arrows || has_bombchu || can_hookshot)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && (short_hook_anywhere || can_dive_small))"
+    "Great Bay Temple Pot Green Pipe 2 6": "(can_dive_small && (has_arrows || has_bombchu || can_hookshot)) || underwater_walking || hookshot_anywhere || (can_use_ice_arrows && (short_hook_anywhere || can_dive_small))"
+    "Great Bay Temple Pot Green Pipe 2 7": "(can_dive_small && (has_arrows || has_bombchu)) || underwater_walking || (can_use_ice_arrows && (short_hook_anywhere || can_dive_small))"
+    "Great Bay Temple Pot Green Pipe 2 8": "(can_dive_small && (has_arrows || has_bombchu)) || underwater_walking || (can_use_ice_arrows && (short_hook_anywhere || can_dive_small))"
```

Comment:
Why exactly does it need to be with ice arrows if arrows alone are already enough?